<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>音乐播放器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>播放列表</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>添加</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/images/316.bmp</normaloff>:/images/images/316.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRemove">
           <property name="text">
            <string>移除</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/images/318.bmp</normaloff>:/images/images/318.bmp</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClear">
           <property name="text">
            <string>清空</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/images/214.bmp</normaloff>:/images/images/214.bmp</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="toolTip">
         <string>播放</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/620.bmp</normaloff>:/images/images/620.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPause">
        <property name="toolTip">
         <string>暂停</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/622.bmp</normaloff>:/images/images/622.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStop">
        <property name="toolTip">
         <string>停止</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/624.bmp</normaloff>:/images/images/624.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPrevious">
        <property name="toolTip">
         <string>前一曲目</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/616.bmp</normaloff>:/images/images/616.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="toolTip">
         <string>下一曲目</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/630.bmp</normaloff>:/images/images/630.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSound">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/images/volumn.bmp</normaloff>:/images/images/volumn.bmp</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderVolumn">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="LabCurMedia">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>无曲目</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderPosition">
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LabRatio">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>进度</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
